editoast/migrations/2024-09-02-132335_replace_mode_ac_with_air_conditioning/up.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE rolling_stock
+SET effort_curves = REGEXP_REPLACE(effort_curves::text, 'AC', 'AIR_CONDITIONING', 'g')::jsonb;
```

Comment:
To be more restrictive in the regex. I did not test it...

```suggestion
SET effort_curves = REGEXP_REPLACE(effort_curves::text, '"AC"', '"AIR_CONDITIONING"', 'g')::jsonb;
```